rllib/algorithms/simple_q/tests/test_repro_simple_q.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+import ray
+from ray.tune import register_env
+import ray.rllib.algorithms.simple_q as simple_q
+from ray.rllib.examples.env.deterministic_envs import DeterministicCartPole
+from ray.rllib.utils.test_utils import check_reproducibilty
+
+
+class TestReproPPO(unittest.TestCase):
```

Comment:
probably need to rename this to q algorithms or something like that.

Reply:
nice catch again :)

Reply:
fixed it in dqn as well.